multi-node/aws/pkg/config/config.go

```diff
@@ -116,7 +120,9 @@ const (
 
 func (c Cluster) Config() (*Config, error) {
 	config := Config{Cluster: c}
-	config.ETCDEndpoints = fmt.Sprintf("http://%s:2379", c.ControllerIP)
+
+	config.EtcdEndpoint = strings.Split(c.EtcdEndpoints, ",")[0]
+	config.EtcdEndpoints = c.EtcdEndpoints
```

Comment:
this is redundant, as the `Config.EtcdEndpoints` field is inherited from the `Cluster` type